.gitignore

```diff
@@ -38,3 +38,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+# Ignore application yml for credentials database.
+/config/application.yml
```

Comment:
Why are you using an `application.yml` file in the first place? I know that's for defining secrets, but shouldn't that require a gem to get it working?